server/src/Models/DB.hs

```diff
@@ -103,6 +103,7 @@ Customer
     avalaraCode AvalaraCustomerCode Maybe
     isAdmin Bool default=false
     verified Bool default=true
+    ephemeral Bool default=false
```

Comment:
Perhaps, instead of storing the flag, we could change `encryptedPassword` to be `Maybe` so that `Nothing` would indicate that the user is ephemeral, and `Just encryptedPass` would indicate that the user actually registered. This would also address a concern I raised in my previous comment regarding a hardcoded password hash for ephemeral customers.

What do you think?